x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/DeleteJobIT.java

```diff
@@ -97,13 +97,12 @@ private void runJob(String jobId, String datafeedId) throws Exception {
             .setAnalysisConfig(analysisConfig)
             .setDataDescription(dataDescription);
 
-        registerJob(job);
-        putJob(job);
+                putJob(job);
```

Comment:
```suggestion
        putJob(job);
```

Reply:
AGH, my find replace broke in some files 🤦 will fix